tests/testScripts/test_upcoming_public_events.py

```diff
@@ -0,0 +1,19 @@
+import config
+import pytest
+
+
+
+"""
+    Testing the upcoming public events
+"""
+
+
+def test_upcoming_public_events_guest(app):
+    app.landing.go_to_site()
+    assert app.landing.get_upcoming_public_event()
+
+
+def test_upcoming_public_events_after_login(app, admin_setup):
```

Comment:
admin_setup inherits app, so, you don't need call app if you have admin_setup
```suggestion
def test_upcoming_public_events_after_login(app, admin_setup):
```

Reply:
Resolved in the new commit